interface/modules/custom_modules/oe-module-comlink-telehealth/src/TelehealthGlobalConfig.php

```diff
@@ -430,11 +443,27 @@ public function renderFooterBox($fldid, $fldarray)
             ,'fldarray' => $fldarray
             ,'verifyInstallationPathUrl' => $this->publicWebPath . 'index.php?action=verify_installation_settings'
             ,'telehealthCvbUrl' => $this->publicWebPath . 'assets/js/src/cvb.min.js'
+            ,'isLocaleConfigured' => $isLocaleConfigured
         ];
 
         return $this->twig->render("comlink/admin/telehealth_footer_box.html.twig", $dataArray);
     }
 
+    private function isLocaleConfigured()
+    {
+        // timezone is not set in the $GLOBALS array oddly, not sure why, check against the database
+        $record = QueryUtils::fetchRecords("SELECT gl_name, gl_index, gl_value FROM globals WHERE gl_name=?", [self::LOCALE_TIMEZONE]);
```

Comment:
Yes I've seen this and meant to track it down.

Reply:
I know where it's occurring, just thought that might have been intentional.  Didn't want to change anything in the globals though so close to the wire here so I just check against globals.